### PR TITLE
fix deconv with groups > 1

### PR DESCRIPTION
**Pull request by uni19**
# Description
 fix deconv with groups > 1
Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.
- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: core/conversion/converters/impl/conv_deconv.cpp [hunk 1]
```diff
@@ -29,36 +28,41 @@ auto conv_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns().patter
...
         if (args[2].IValue()->isTensor()) {
-          Weights b(ctx, args[2].IValue()->toTensor());
-          deconv = ctx->net->addDeconvolutionNd(*in, w.num_input_maps, w.kernel_shape, w.data, b.data);
+          bias = Weights(ctx, args[2].unwrapToTensor());
         } else {
-          deconv = ctx->net->addDeconvolutionNd(*in, w.num_input_maps, w.kernel_shape, w.data, {});
+          bias = Weights(ctx, torch::zeros(args[1].unwrapToTensor().sizes()[1] * groups));
         }
 
+        auto deconv = ctx->net->addDeconvolutionNd(*in, w.num_input_maps * groups, w.kernel_shape, w.data, bias.data);
```
Comment (peri044) [inline core/conversion/converters/impl/conv_deconv.cpp:42]: Can you explain a bit about w.num_input_maps * groups and maybe mention it as a comment ?

**Review by peri044 — CHANGES_REQUESTED** [review]
Posted my comment. The code LGTM.

File: core/conversion/converters/impl/conv_deconv.cpp (hunk 1 above)
Comment (uni19) [reply core/conversion/converters/impl/conv_deconv.cpp]: updated

**Review by narendasan — APPROVED** [review]
LGTM, @peri044 can you take a look and see if the comments are good for you?

**Review by peri044 — APPROVED** [review]

**Comment by peri044** [comment]
LGTM

**Pull request merged**